src/frequenz/dispatch/_dispatcher.py

```diff
@@ -60,8 +81,19 @@ async def run():
             service_address = "localhost:50051"
             dispatcher = Dispatcher(microgrid_id, grpc_channel, service_address)
             dispatcher.start()  # this will start the actor
-            events = dispatcher.lifecycle_events.new_receiver()
-            ready = dispatcher.ready_to_execute.new_receiver()
+
+            events_receiver = dispatcher.lifecycle_events.new_receiver()
+
+            async for event in events_receiver:
```

Comment:
This makes me think we should also expose the client as `dispatcher.client` so the user can react to the events in a meaningful way when they want to update or create a new event as a response

Reply:
I don't get this. Why do you need the client?

Reply:
Well, anyone who wants to update, delete or create a request needs the client to do so, the high-level API 
doesn't offer those things (and I don't see how it could do that better than the client)